test/views/YourSelfAssessmentChargesViewSpec.scala

```diff
@@ -75,8 +75,8 @@ class YourSelfAssessmentChargesViewSpec extends TestSupport with FeatureSwitchin
   val poa2Text: String = messages("selfAssessmentCharges.paymentOnAccount2.text")
   val poaExtra1Text: String = messages("selfAssessmentCharges.reviewAndReconcilePoa1.text")
   val poaExtra2Text: String = messages("selfAssessmentCharges.reviewAndReconcilePoa2.text")
-  val poa1ReconcileInterest: String = messages("selfAssessmentCharges.lpi.reviewAndReconcilePoa1.text")
-  val poa2ReconcileInterest: String = messages("selfAssessmentCharges.lpi.reviewAndReconcilePoa2.text")
+  val poa1ReconcileIreviewAndReconcilenterest: String = messages("selfAssessmentCharges.lpi.reviewAndReconcilePoa1.text")
+  val poa2ReconcileInterest: String = messages("selfAssessmentCharges.lpi.Poa2.text")
```

Comment:
The message key shouldn't change, the charges still exist and we still want to call the message for reviewAndReconcilePoa2